closed/autoconf/custom-hook.m4

```diff
@@ -394,7 +396,7 @@ AC_DEFUN([CONFIGURE_OPENSSL],
       [enable bundling of the openssl crypto library with the jdk build])])
    WITH_OPENSSL=yes
    if test "x$with_openssl" = x; then
-    # User doesn't want to build with OpenSSL.. Ensure that jncrypto library is not built
+    # User doesn't want to build with OpenSSL.. No need to build openssl libraries.
```

Comment:
nit: please remove the duplicate period.

Reply:
Thank you @keithc-ca. Done.